src/applications/simple-forms/21-4142/pages/personalInformation2.js

```diff
@@ -19,12 +17,23 @@ export default {
       [veteranFields.vaFileNumber]: {
         'ui:title': 'VA file number (if applicable)',
         'ui:errorMessages': {
-          pattern:
-            'Please input a valid VA file number: 7 to 9 numeric digits, & may start with a letter "C" or "c".',
+          pattern: 'Your VA file number must be 8 or 9 digits',
+        },
+        'ui:options': {
+          replaceSchema: () => {
+            return {
+              type: 'string',
+              pattern: '^\\d{8,9}$',
```

Comment:
for all the vaFileNumbers, there is also the pattern that starts with [cC]{0,1} in common definitions. did we want to use that, or this more loose pattern?

Reply:
We can't use that one because central mail won't accept that pattern in their metadata, so 8-9 is the only thing we can allow for now